oauth-auth-code/csharp/Controllers/HomeController.cs

```diff
@@ -1,93 +1,97 @@
-﻿using System;
+﻿using authorization_code_aspdotnet.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
 using System.Collections.Generic;
-using System.IO;
+using System.Diagnostics;
 using System.Linq;
+using System.Threading.Tasks;
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Text;
-using System.Threading.Tasks;
-using Microsoft.AspNet.Mvc;
-using Microsoft.Framework.WebEncoders;
+using System.Text.Encodings.Web;
 using Newtonsoft.Json.Linq;
 
 namespace authorization_code_aspdotnet.Controllers
 {
     public class HomeController : Controller
     {
+        private readonly ILogger<HomeController> _logger;
         private string host = "mypurecloud.com";
```

Comment:
Might be best to indicate with a comment or otherwise that users might need to update the host and rediretUri to avoid confusion. If it's already documented somewhere then that's fine.

Reply:
+Added comment